src/yardstick/tool/grype.py

```diff
@@ -286,31 +331,38 @@ def install(
                 src_path=version.removeprefix("path:"),
                 version=version.removeprefix("path:"),
                 use_cache=use_cache,
+                db_identity=db_identity,
                 profile=grype_profile,
+                add_version_to_path=add_version_to_path,
                 **kwargs,
             )
         else:
             tool_obj = cls._install_from_git(
                 version=version,
                 path=path,
+                db_identity=db_identity,
                 use_cache=use_cache,
                 profile=grype_profile,
+                add_version_to_path=add_version_to_path,
                 **kwargs,
             )
 
         # always update the DB, raise exception on failure
         if db_import_path:
-            logging.debug(f"using given db from {db_import_path!r}")
-            tool_obj.run("db", "import", db_import_path)
+            if os.path.exists(tool_obj.db_root):
+                logging.info(f"using existing (custom) db from {tool_obj.db_root!r}")
+            else:
+                logging.info(f"importing given (custom) db from {db_import_path!r}")
+                tool_obj.run("db", "import", db_import_path)
         elif update_db:
-            logging.debug("updating db")
+            logging.debug("updating db from OSS")
             tool_obj.run("db", "update")
 
         return tool_obj
 
     @property
     def db_root(self) -> str:
-        return os.path.join(self.path, "db")
+        return os.path.join(self.path, "db", self.db_identity)
```

Comment:
So this will just have `oss` in it for grype installs that are using the normal database?

Reply:
yup 👍